utils/FormatPetsAge.jsx

```diff
@@ -0,0 +1,14 @@
+export default function FormatPetsAge(seconds) {
```

Comment:
function name is capitalized, but this is no function component, also I would suggest to have a `/utils/index.js` where you can store multiple helper functions

Reply:
Done, thanks!